BitwardenShared/UI/Tools/Send/SendItem/AddEditSendItem/AddEditSendItemView.swift

```diff
@@ -32,8 +32,10 @@ struct AddEditSendItemView: View { // swiftlint:disable:this type_body_length
                     VStack(alignment: .leading, spacing: 16) {
                         if store.state.isSendDisabled {
                             InfoContainer(Localizations.sendDisabledWarning)
+                                .accessibilityIdentifier("DisabledSendPolicyLabel")
```

Comment:
🤔 I think we should be consistent with the A11y IDs on the `InfoContainer` suffixes.
I see the one in `SendListView` is called `...CalloutLabel` whereas in here it's `...PolicyLabel` and it's the same warning displayed. Also not sure why the name before the suffix changes for the same warning.

Reply:
We decided to use a different ending for those IDs because the policy message we are displaying now on `SendListView` might be different in the future. 
But I agree with you on the consistency, so I'm updating it on my next commit. 

Thank you!